### ISSUE TITLE
Request await won't end

### ISSUE THREAD

**Issue by xiaodong2008** [fastjs-team/core#129]
### What is the current behavior?

```javascript
await request.get('https://reqres.in/api/users').then(() => {
  console.log("then")
});
console.log("end")
```
It will only log then

### What is the expected behavior?

It should log then, then output end

### Steps to reproduce the problem

Upgrade your version to `v1.0.0-alpha.10`, ~~problem starts at this version~~.

### Link to minimal reproduction

https://stackblitz.com/edit/fastjs-reproduce-zrvk6a?file=src/main.ts

### Confirmations

- [X] I'm using/tried the latest version of fastjs and the bug is still there
- [X] I have read the [Code of Conduct](https://docs.fastjs.dev/other/code-of-conduct.html)
- [X] I have read the [Issue Reporting Guidelines](https://github.com/fastjs-team/fastjs-next/blob/main/.github/contributing.md#issue-reporting)
- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [X] I know that any inappropriate content may cause the maintainer to take action base on the [Code of Conduct](https://docs.fastjs.dev/other/code-of-conduct.html)

**Comment by xiaodong2008** [comment]
Hmm, it seems like the problem starts at `v1.0.0-alpha.12` -> `v1.0.0-alpha.13`.

**Comment by xiaodong2008** [comment]
Fixed: 0896235e4fb07aaf1ffe87e336bf52a68f2c3e33
